.gitignore

```diff
@@ -0,0 +1 @@
+coverage.out
```

Comment:
I added this since I was generating coverage via

```
rm -f coverage.out
go test -coverprofile=coverage.out ./...
go tool cover -html=coverage.out
```